_regional/djangocon-africa.yml

```
@@ -0,0 +1,6 @@
+---
+name: DjangoCon Africa
+location: Arusha, Tanzania
```

Since this is a rotating event, and to prevent having to update this annually, we should use a similar format to other rotating location events (like DjangoCon US): 

```suggestion
location: Africa (rotating)
```